great_expectations/agent/actions/data_assistants/utils.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime
+from typing import TYPE_CHECKING
+
+from great_expectations.agent.actions.agent_action import (
+    ActionResult,
+)
+from great_expectations.agent.models import (
+    CreatedResource,
+    RunDataAssistantEvent,
+)
+from great_expectations.datasource.fluent import Datasource as FluentDatasource
+
+if TYPE_CHECKING:
+    from great_expectations.core import ExpectationSuite
+    from great_expectations.core.batch import BatchRequest
+    from great_expectations.rule_based_profiler.data_assistant_result.data_assistant_result import (
+        DataAssistantResult,
+    )
+    from great_expectations.data_context.data_context import CloudDataContext
+
+
+def build_batch_request(
+    context: CloudDataContext,
```

Comment:
Are you able to run this successfully on your end? this is raising an exception for me locally - i don't think it can be in a type checking blog.